.circleci/config.yml

```diff
@@ -13,30 +13,4 @@ jobs:
       - run:
           name: "Enforce Go Formatted Code"
           command: "! go fmt ./... 2>&1 | read"
```

Comment:
Any reason not to replace this with `make fmt-check`?

Reply:
Ok done.